caddyhttp/httpserver/logger_test.go

```diff
@@ -20,13 +20,13 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
+	"log"
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
 	"testing"
 
-	syslog "gopkg.in/mcuadros/go-syslog.v2"
```

Comment:
This line should be back. It's odd that it got removed while still referenced throughout the code.